### ISSUE TITLE
make shortcut button and submenu accessible (#7833 reopened)

### ISSUE THREAD

**Issue by robfentress** [storybookjs/storybook#9451]
Issue [#7833](https://github.com/storybookjs/storybook/issues/7833) is still a bug (it is not really a feature request, since it presents a real barrier for some people).  See the [Web Content Accessibility Guidelines Success Criteria 2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/quickref/#keyboard) for details on why the current implementation is problematic, if you are unsure.  I would like to be able to recommend the use of Storybook to others in our organization, because it is awesome, but cannot, because we have made a commitment to equal access for people with disabilities.

A good example of one way to handle the main menu functionality can be found in the [Cauldron](https://pattern-library.dequelabs.com/) Library's [Menu Navigation Composite Widget](https://pattern-library.dequelabs.com/composites/menu-navigation).  For an example of a good way to handle the shortcut button, see Cauldron's [Option Menus Component](https://pattern-library.dequelabs.com/components/option-menus).

**Comment by shilman** [comment]
@robfentress Any interest in giving this a shot? PR's very welcome!

**Comment by robfentress** [comment]
Oh no, I was afraid you were gonna say that!  :-)

I'd love to, but, realistically, I don't see it happening anytime soon.  When I get a free minute, I'll look at the code to see how hard it would be.

**Comment by robfentress** [comment]
Hey, @mikesir87.  Any interest in taking this on?
